lib/internal/errors.js

```diff
@@ -1104,18 +1105,28 @@ E('ERR_INVALID_PACKAGE_CONFIG', (path, message, hasMessage = true) => {
 }, Error);
 E('ERR_INVALID_PACKAGE_TARGET',
   (pkgPath, key, subpath, target, base = undefined) => {
+    const relError = typeof target === 'string' &&
+      target.length && !StringPrototypeStartsWith(target, './');
     if (key === null) {
       if (subpath !== '') {
         return `Invalid "exports" target ${JSONStringify(target)} defined ` +
         `for '${subpath}' in the package config ${pkgPath} imported from ` +
-        base;
+        `${base}.${relError ? ' - targets must start with "./"' : ''}`;
       } else {
         return `Invalid "exports" main target ${target} defined in the ` +
-          `package config ${pkgPath} imported from ${base}.`;
+          `package config ${pkgPath} imported from ${base}.${relError ?
```

Comment:
@DerekNonGeneric good point... would this be the suggestion then?

```suggestion
          `package config ${pkgPath} imported from ${base}${relError ?
```

Reply:
Yep, that's what I was thinking.

Reply:
If you wanted to be perfectly correct, that should be an em dash (`—`) if not for the [`non-ascii-character` lint rule](https://github.com/nodejs/node/blob/master/tools/eslint-rules/non-ascii-character.js), which would also prevent using curly quotes as suggested by @ljharb. Some people like to just use two consecutive dashes (`--`) in it's place, but I've never been a fan. :p